src/scripts/hero-animation/rAF.js

```diff
@@ -0,0 +1,31 @@
+// http://paulirish.com/2011/requestanimationframe-for-smart-animating/
```

Comment:
I think we can safely ditch this `rAF.js` file. It's a `reuqestAnimationFrame()` polyfill, but native browser support is now broad enough that we can do without it. Besides, even if an ancient browsers is used to view the site, it's only the animation that will fail. Since it's not critical page content, I think that's acceptable.

Also, our `smoothScroll` code also uses `reuqestAnimationFrame()` directly, so there's a precedent for omitting the polyfill. :-)